bokehjs/src/coffee/models/annotations/color_bar.coffee

```diff
@@ -515,6 +522,7 @@ export class ColorBar extends Annotation
       minor_coords[i] = new Float64Array((scale_length - coord for coord in minor_coords[i]))
 
     return {
+      "major_ticks": major_ticks
       "major": major_coords
       "minor": minor_coords
       "major_labels": major_labels
```

Comment:
This is getting a little cluttered. I think if we are touching this code it would be a good time to clean it up a bit with some light refactoring, e.g. perhaps return instead
```
return {
  ticks: ticks
  coords: coords
  labels: labels
```

Where each of the above has `.major` and (optionally) `.minor` as attributes. 

I think it's also worth considering applying the formatting (including label overrides) here, in this function, rather than in the view. In any case the view is calling `_tick_coordinates()` twice! We should definitely fix that since it's unnecessarily wasteful. I also think these methods could be made public.